cmd/cluster-operator/options/options.go

```diff
@@ -110,6 +113,13 @@ func (opt *Options) AddFlags(fs *pflag.FlagSet) {
 		"Number of Cluster API resources to process simultaneously",
 	)
 
+	fs.DurationVar(
+		&opt.PollInterval,
+		"poll-interval",
```

Comment:
Prefer not to call `poll`, it would mislead suers to think we are working on polling.

Maybe `requeue-after`

Reply:
done